### PR TITLE
workflows: remove out of date and fix

### PR DESCRIPTION
**Pull request by patrick-stephens**
Repository included out of date file - all files are on the releases themselves.

The workflows were also updated.

### CODE REVIEW

File: .github/workflows/lint.yaml [hunk 1]
```diff
@@ -26,5 +19,6 @@ jobs:
     runs-on: ubuntu-latest
     name: PR - Markdownlint
     steps:
+      - uses: actions/checkout@v4
```
Comment (patrick-stephens) [inline .github/workflows/lint.yaml:22]: Without this we were doing nothing... :(

Comment (jasmingacic) [reply .github/workflows/lint.yaml:22]: I'm not sure I understand

Comment (patrick-stephens) [reply .github/workflows/lint.yaml:22]: Nothing is checked out so nothing is actually linted, prior to this change.

You can see we actually exceed the line length rule but this was not picked up before.

**Review by jasmingacic — APPROVED** [review]
looks legit

**Pull request merged**